bookshelf/web/index.php

```diff
@@ -18,14 +18,20 @@
 namespace Google\Cloud\Samples\Bookshelf;
 
 use Google\Cloud\Samples\Bookshelf\DataModel\CloudSql;
+use Google\Cloud\Samples\Bookshelf\DataModel\Datastore;
 use Google\Cloud\Samples\Bookshelf\FileSystem\CloudStorage;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
 $app = require __DIR__ . '/../src/app.php';
 require __DIR__ . '/../src/controllers.php';
-$app['bookshelf.model'] = new CloudSql();
+
+// Cloud Storage
 $bucket = getenv('GOOGLE_STORAGE_BUCKET');
 $app['bookshelf.storage'] = new CloudStorage($bucket);
+// Cloud SQL
+$app['bookshelf.model'] = new CloudSql();
+// Cloud Datastore
+// $app['bookshelf.model'] = new Datastore($bucket);
```

Comment:
I noticed that you're passing $bucket to the Datastore model. Let's make it an env var.
Also how about to decide the data model backend with another env var?